stdlib/Test/docs/src/index.md

```diff
@@ -322,8 +322,12 @@ function finish(ts::CustomTestSet)
     # just record if we're not the top-level parent
     if get_testset_depth() > 0
         record(get_testset(), ts)
+        return ts
     end
-    ts
+
+    # so the results are printed if we are at the top level
+    Test.print_test_results(ts)
+    return ts
```

Comment:
Minor but this might be clearer as
```
    if get_testset_depth() > 0
        # just record if we're not the top-level parent
        record(get_testset(), ts)
    else
        # at the top level print the results
        Test.print_test_results(ts)
    end
    return ts
```

Reply:
If a custom testsets supports the `verbose` setting, the `else` version would become more complicated. With the existing code, it's one nesting level less.